packages/react-conformance/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-conformance",
-  "version": "0.1.5",
+  "version": "1.0.0-beta",
```

Comment:
Not sure this one is quite at version 1 yet either...but also I don't know how likely it is that anyone would be using it